apps/client/src/views/cuesheet/CuesheetPage.tsx

```diff
@@ -106,6 +120,7 @@ export default function CuesheetPage() {
         data={flatRundown}
         columns={columns}
         handleUpdate={handleUpdate}
+        handleUpdateCustom={handleUpdateCustom}
         selectedId={featureData.selectedEventId}
         currentBlockId={featureData.currentBlockId}
```

Comment:
these will change pretty often, I wonder if this is an easy point for improvement

The `currentBlockId` is only used in the rows, we could create a row element that gets this data from the socket
We could further stabilise this by creating a selector on whether the `currentBlockIsSelected`

the `selectedId` is used to make `isPast` and a bit trickier to stabilise but it could be worth it

We shouldnt do that now but keep in mind for a near future improvement task